### ISSUE TITLE
Cannot start VM

### ISSUE THREAD

**Issue by JulianGro** [quickemu-project/quickgui#77]
On Debian Testing I cannot start a VM.
So I got Quickget, Quickemu, and QuickGUI. Start QuickGUI, change the path, download Ubuntu 18.04 inside QuickGUI, then it just says "cancel".  When going back and into the VM manager inside QuickGUI, there is an Ubuntu 18.04 VM. Pressing start on that makes the buttons change colours as if it was running and then change back to normal. Nothing seems to happen. Nothing is printed to console.
QuickGUI created a folder for ubuntu containing the iso file and an ubuntu-18.04.conf containing:
```yaml
guest_os="linux"
disk_img="ubuntu-18.04/disk.qcow2"
iso="ubuntu-18.04/ubuntu-18.04.6-desktop-amd64.iso"
```
The mentioned "ubuntu-18.04/disk.qcow2" does not exist.

**Comment by JulianGro** [comment]
Okay so I found some problems for why this doesn't start.
First of all there is no error messages or passthrough of console output, so I was actually missing qemu.
Now it still doesn't start.
Running `quickemu --vm ubuntu-18.04.conf` yields a `qemu-system-x86_64: Display spice is incompatible with the GL context`
Not sure what that means because doing `quickemu --vm ubuntu-18.04.conf --display spice` works fine.

**Comment by flexiondotorg** [comment]
This was fixed in Quickemu 4.1: https://github.com/quickemu-project/quickemu/releases/tag/4.1
